app/jobs/children_support_module/verify_available_modules_task_job.rb

```diff
@@ -6,6 +6,8 @@ def perform(group_id)
       logistics_team_members = AdminUser.all_logistics_team_members
       child_support_link = {}
       group.children.each do |child|
+        next unless child.child_support
```

Comment:
Enregistrer les cas repérés de la sorte : 
errors << "Ceci est une erreur" and next if child.child_support.blank?